docs/api.rst

```diff
@@ -0,0 +1,249 @@
+.. _ksql-http-api:
+
+HTTP API Reference
```

Comment:
also, i'd suggest renaming to `KSQL HTTP API Reference` - so that search results will display better.